setup.py

```diff
@@ -20,11 +22,9 @@
     install_requires=[
         'numpy',
         'pandas',
-        'py4j',
-        'pyarrow>=0.8.0',
-        'pyspark',
         'scipy',
         'statsmodels>=0.9',
+        'pyarrow>=0.8.0',
```

Comment:
where is pyarrow being used? i don't see it being import anywhere

Reply:
cf https://github.com/mozilla/mozanalysis/pull/38/files/9f79049305a165e367ad2bea912d31533bf89b8e#diff-2eeaed663bd0d25b7e608891384b7298 -- pyspark will vomit if you try to define a pandas udf without a recentish pyarrow